docs/run-platform-server/ingestion-filtering.mdx

```diff
@@ -97,29 +58,12 @@ I would like to store the full history of all transactions related from the gene
 
 ### Pre-requisites:
 
-You have an existing Horizon installed, configured and has forward ingestion enabled at a minimum to be able to successfully sync to the current state of the Stellar network. Bonus if you are familiar with running re-ingestion.
-
-Steps:
-
-1. Configure 4 whitelisted Assets via the Admin API. Also check the `HISTORY_RETENTION_COUNT` and set it to `0` if you don’t want any history purged anymore now that you are filtering, otherwise it will continue to reap all data older than the retention.
-
-2. Decide if you want to wipe existing history data on the DB first before the filtering starts running, you can effectively clear the history by running
-
-```
-HISTORY_RETENTION_COUNT=1 stellar-horizon db reap
-```
-
-or drop/create the db and run `stellar-horizon db init`.
-
-Alternatively, if you do not need to free up old history tables, you can effectively stop here, anytime changes or enablement of filter rules are done, the history tables will immediately reflect filtered data per those latest rules from the time the filter config is updated and forward.
-
-3. If starting with a fresh DB, decide if you want to re-run ingestion from the earliest ledger # related to the whitelisted entities to populate history for just the allowed data from filters.
+You have installed Horizon with empty database, it has **live** ingestion enabled.
 
-- Tip: To find this ledger number, you can check for the earliest transaction of the Account issuing that asset.
-- Also consider running parallel workers to speed up the process.
+### Steps:
 
-4. Optional: When re-ingestion is finished, run an ingestion gap fill `stellar-horizon db fill-gaps` to fill any gaps that may have been missed.
+1. Configure a filter rule with 4 whitelisted Assets via the Admin API. Also check the `HISTORY_RETENTION_COUNT` and set it to `0` if you don’t want any history purged anymore now that you are filtering, otherwise it will continue to reap all data older than the retention.
```

Comment:
```suggestion
1. Configure a filter rule with 4 whitelisted Assets via the Admin API
```
I would just leave this part out. Someone could in theory want to retain limited history *and* use ingestion filtering. And think we should just overall avoid plugging full history use cases

Reply:
yes, will drop, it's just a compelling side effect of filtering that wanted to highlight it as a way to encourage it's usage moreso.